content/process/repositories.md

```diff
@@ -116,6 +116,16 @@ Reviewers should only block a PR by `Requesting Changes` if one or more elements
 
 Once a PR has been reviewed and approved by at least one reviewer, it can be merged. The only way we allow merging is by squashing all the PR's commits into a single one, as this keeps the `master` branch and the changelog clean and easy to follow.
 
+!!! warning "Closing Pull Requests"
+
+    When closing a Pull Request, make sure to always specify the reason for closing it by adding a comment to the PR while closing it.
```

Comment:
```suggestion
    When a PR needs to be closed, please make sure to always specify the reason for this action in a comment to the PR along with the closing.
```

Reply:
```suggestion
    When closing a Pull Request, make sure to always specify the reason for doing so by adding a comment to the PR.
```

Reply:
Yes, it works, but it is too conversational for a written text, especially in a guideline.
But anyway.